src/amo/reducers/users.js

```diff
@@ -272,6 +336,24 @@ const reducer = (
   action: Action
 ): UsersStateType => {
   switch (action.type) {
+    case CANCEL_EDIT_USER_ACCOUNT: {
+      const { userId } = action.payload;
+
+      return {
```

Comment:
As an example of what I meant above, instead of calling a `setUserAsNotEditing` helper here, you could just make this:

```js
      return {
        ...state,
        isEditingById: {...state.isEditingById, [userId]: false },
      };
```

Reply:
Fair enough. I think I'm just growing to like using selectors and helpers to consolidate this kind of behaviour, but this one is pretty simple. I'll see about removing them.